plugins/system/cpu.go

```diff
@@ -4,10 +4,22 @@ import (
 	"fmt"
 
 	"github.com/influxdb/telegraf/plugins"
+	"github.com/influxdb/telegraf/plugins/system/ps/cpu"
 )
 
 type CPUStats struct {
-	ps PS
+	ps        PS
+	lastStats []cpu.CPUTimesStat
+}
+
+func NewCPUStats(ps PS) *CPUStats {
```

Comment:
I know a lot of the plugins don't conform to this at the moment (this is something I would like to improve), but all exported functions should be commented